ci-jobs/scripts/build-webdriveragent.js

```diff
@@ -0,0 +1,49 @@
+const path = require('path');
+const os = require('os');
+const { asyncify } = require('asyncbox');
+const { logger, fs, mkdirp } = require('appium-support');
+const { exec } = require('teen_process');
+const xcode = require('appium-xcode');
+
+const log = new logger.getLogger('WDABuild');
+
+async function buildWebDriverAgent (xcodeVersion) {
+  // Get Xcode version
+  xcodeVersion = xcodeVersion || await xcode.getVersion();
+  log.info(`Building bundle for Xcode version '${xcodeVersion}'`);
+
+  // Clean and build
+  await exec('npx', ['gulp', 'clean:carthage']);
+  log.info('Running ./Scripts/build.sh');
+  await exec('./Scripts/build.sh');
```

Comment:
does it work like this? I thought it should be done like `exec('/bin/bash', ['./Scripts/build.sh'], {cwd: ...})`

Reply:
It does work but I'll change it anyway to be safe.